app/instance/.env.example

```
@@ -15,14 +15,32 @@ SUPABASE_REST_API=
 SUPABASE_HOST=
 SUPABASE_USER=
 SUPABASE_PASSWORD=
-SUPABASE_PORT=
+SUPABASE_PORT=6543 #use Transaction shared pooler for db and session pooler for queue
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SUPABASE_PORT key should go before the SUPABASE_USER key

(UnorderedKey)